.travis.yml

```diff
@@ -7,4 +7,4 @@ install:
     - pip install -r requirements.txt
 
 script:
-    ./test_reframe.py
```

Comment:
We will need to be able to run the the tests with `./test_reframe.py` passing any options to pytest. We can achieve this by calling `pytest.main()` instead of `nose.main` inside the `test_reframe.py`.

Reply:
Yes, I know but first we must add also pytest to corresponding Python bare module.

Reply:
In any case we should add it to the `requirements.txt`, so that anybody can run the unit tests.